src/Sarif/FileRegionsCache.cs

```diff
@@ -92,6 +105,158 @@
             this._cache.Clear();
         }
 
+        public void SuvamTest(
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

mark things as internal if you need them for tests.

Reply:
@michaelcfanning nuked this